electrode_models/nucleation_single_particle/initialize.py

```diff
@@ -64,24 +70,18 @@ class electrode:
 
         # Determine the electrode capacity (Ah/m2)
 
-        # Max concentration of stored ion (intercalated Li)
-        # Save initial X
-        X_o = bulk_obj.X 
-        # Set object concentration to fully lithiated:
-        bulk_obj.X = inputs['stored-ion']['name']+':1.0' 
-        # Concentration of stored Li, per unit volume of intercalation phase:
-        C = bulk_obj[inputs['stored-ion']['name']].concentrations[0]
-        
-        capacity = (C*inputs['stored-ion']['charge']*ct.faraday
-                *inputs['eps_solid'])*inputs['thickness']/3600
+        capacity = (inputs['stored-species']['charge']*ct.faraday
```

Comment:
The charge and MW for this species are already provided to Cantera... no need to make the user provide them twice.  I'll probably accept this as is, and then we can replace this with some Cantera function calls.

Reply:
Lol, I think this is basically the same approach I am already using, elsewhere.  Please leave as is... 😂

Reply:
Yeah, I'm unsure of how to call some of those cantera properties I think there's a few places where I mentioned that in the comments maybe we can go over it.